src/get-started/operate-iroha-2-via-cli.md

```diff
@@ -348,14 +322,68 @@ Listening to events with filter: Pipeline(Block(BlockEventFilter { height: None,
   "Pipeline": {
     "Block": {
       "header": {
-        "height": 14,
-        "prev_block_hash": "AF1ABC889019971D4C4E8866C347367D63A024319E50AEF989DB255F761E9D1D",
-        "transactions_hash": "7F2091D887BF9DBF6100DFEA696B06AE269C288AE55F1D281D9FDDAD93D1B8F1",
-        "creation_time_ms": 1721132667162,
-        "view_change_index": 1,
-        "consensus_estimation_ms": 4000
+        "height": 17,
+        "prev_block_hash": "0F6B562D8B3F635C46CD7BEB20C801B8C7594B56AD5F3E0CFBD596C0894885F3",
+        "transactions_hash": "5A85B662672155A7D93DC54A132975FA4252152AB04CA224CE49C7FF97D742AB",
+        "creation_time_ms": 1742405889662,
+        "view_change_index": 2
+      },
+      "status": "Created"
+    }
+  }
+}
+{
+  "Pipeline": {
+    "Block": {
+      "header": {
+        "height": 17,
+        "prev_block_hash": "0F6B562D8B3F635C46CD7BEB20C801B8C7594B56AD5F3E0CFBD596C0894885F3",
+        "transactions_hash": "5A85B662672155A7D93DC54A132975FA4252152AB04CA224CE49C7FF97D742AB",
+        "creation_time_ms": 1742405889662,
+        "view_change_index": 2
+      },
+      "status": "Approved"
+    }
+  }
+}
+{
+  "Pipeline": {
+    "Block": {
+      "header": {
+        "height": 17,
+        "prev_block_hash": "0F6B562D8B3F635C46CD7BEB20C801B8C7594B56AD5F3E0CFBD596C0894885F3",
+        "transactions_hash": "5A85B662672155A7D93DC54A132975FA4252152AB04CA224CE49C7FF97D742AB",
+        "creation_time_ms": 1742405889662,
+        "view_change_index": 2
+      },
+      "status": "Committed"
+    }
+  }
+}
+{
+  "Pipeline": {
+    "Block": {
+      "header": {
+        "height": 17,
+        "prev_block_hash": "0F6B562D8B3F635C46CD7BEB20C801B8C7594B56AD5F3E0CFBD596C0894885F3",
+        "transactions_hash": "5A85B662672155A7D93DC54A132975FA4252152AB04CA224CE49C7FF97D742AB",
+        "creation_time_ms": 1742405889662,
+        "view_change_index": 2
+      },
+      "status": "Applied"
+    }
+  }
+}
+{
+  "Pipeline": {
+    "Block": {
+      "header": {
+        "height": 18,
```

Comment:
As an example output, events for height 17 would be enough

Reply:
updated in commit 894fe1e2024db529371d0d0ba9f1fecea2a56a06